gendox-core-api/src/main/java/dev/ctrlspace/gendox/gendoxcoreapi/services/UserOrganizationService.java

```diff
@@ -28,4 +53,40 @@ public List<UserOrganization> getAll(UserOrganizationCriteria criteria, Pageable
         return userOrganizationRepository.findAll(UserOrganizationPredicate.build(criteria), pageable).toList();
 
     }
+
+    public void setAdminRoleForOrganizationsOwner(Organization organization) {
+        UserOrganization userOrganization = new UserOrganization();
+        Instant now = Instant.now();
+
+        // user
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        JwtDTO jwtDTO = jwtUtils.toJwtDTO((Jwt)authentication.getPrincipal());
+        String userIdString = jwtDTO.getUserId(); // Assuming you have the UUID as a string
+        UUID userId = UUID.fromString(userIdString);
+        User user = userRepository.getById(userId);
+
+        // role
+        Type role = typeRepository.getById(3L);
+
+        userOrganization.setUser(user);
+        userOrganization.setOrganizationId(organization);
+        userOrganization.setRole(role);
+        userOrganization.setCreatedAt(now);
+        userOrganization.setUpdatedAt(now);
+
+        userOrganizationRepository.save(userOrganization);
```

Comment:
Refactor the above to something like this:

```java

       userOrganization.setUser(user);
        userOrganization.setOrganizationId(organization);
        userOrganization.setRole(role);
        
        userOrganizationService.create(userOrganization);
```
In the userOrganizationService.create:
```java

        userOrganization.setCreatedAt(now);
        userOrganization.setUpdatedAt(now);
        
        userOrganizationRepository.save(userOrganization);
```

This way that code will be more reusable